src/components/Widgets/Clock.vue

```diff
@@ -37,10 +37,14 @@ export default {
       return this.timeZone.split('/')[1].replaceAll('_', ' ');
     },
     showSeconds() {
-      return !this.options.hideSeconds;
+      if (this.options.hideSeconds) return !this.options.hideSeconds;
+      // this is the default
+      return true;
```

Comment:
Since the `!` is converting to a bool as well as negating, there shouldn't be a need to explicitly return `true`

Reply:
I know, but its more cleaner and the default is clearly visible, the main question here is, can this defaukt change, here I think it wan't so I agree that the ! is cleaner.